R-proj/examples/logconcave/generalized_hyperbolic.r

```diff
@@ -0,0 +1,65 @@
+# VolEsti (volume computation and sampling library)
```

Comment:
For consistency please use the same file extension with capital R as the other files